### PR TITLE
Added Norwegian localization

### PR DESCRIPTION
**Pull request by gustaf08**

### CODE REVIEW

**Comment by erikng** [comment]
I don't think this will work as is because it was likely created outside of Xcode. The Xcode project doesn't know where to add it to the app bundle. 

Typically if you look at other PRs for languages, two files are modified.

**Comment by gustaf08** [comment]
Hopefully its fixed now.

File: Nudge.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -366,6 +367,8 @@
 				pt,
 				hi,
 				ru,
+				no,
+				nb,
```
Comment (erikng) [inline Nudge.xcodeproj/project.pbxproj:371]: Where did this come from?

**Review by erikng — CHANGES_REQUESTED** [review]

**Comment by gustaf08** [comment]
Sorry am a bit new to this, have updated it.

**Comment by erikng** [comment]
Screen Shot 2022-02-22 at 8 36 33 AM
Screen Shot 2022-02-22 at 8 36 24 AM
Looks good to me

**Pull request merged**